service/src/test/java/uk/gov/hmcts/reform/fpl/service/ChildrenServiceTest.java

```diff
@@ -1,364 +1,288 @@
-        service.addPageShowToCaseDetails(caseDetails, children);
-
-        assertThat(caseDetails.getData()).extracting("pageShow").isEqualTo("No");
-    }
-
-    private Element<Child> childWithDetailsHiddenNo(UUID id) {
-        return Element.<Child>builder()
-            .id(id)
-            .value(Child.builder()
-                .party(ChildParty.builder()
-                    .firstName("James")
-                    .detailsHidden("No")
-                    .email(EmailAddress.builder().email("email@email.com").build())
-                    .address(Address.builder()
-                        .addressLine1("Address Line 1")
-                        .build())
-                    .telephoneNumber(Telephone.builder().telephoneNumber("01227 831393").build())
-                    .build())
-                .build())
-            .build();
-    }
-
-    private Element<Child> childWithRemovedConfidentialFields(UUID id) {
-        return Element.<Child>builder()
-            .id(id)
-            .value(Child.builder()
-                .party(ChildParty.builder()
-                    .firstName("James")
-                    .detailsHidden("Yes")
-                    .build())
-                .build())
-            .build();
-    }
-
-    private Element<Child> childWithConfidentialFields(UUID id) {
-        return Element.<Child>builder()
-            .id(id)
-            .value(Child.builder()
-                .party(ChildParty.builder()
-                    .firstName("James")
-                    .detailsHidden("Yes")
-                    .email(EmailAddress.builder().email("email@email.com").build())
-                    .address(Address.builder()
-                        .addressLine1("Address Line 1")
-                        .build())
-                    .telephoneNumber(Telephone.builder().telephoneNumber("01227 831393").build())
-                    .build())
-                .build())
-            .build();
-    }
-
-    private List<Element<Child>> childElementWithDetailsHiddenValue(String hidden) {
-        return ImmutableList.of(Element.<Child>builder()
-            .id(randomUUID())
-            .value(Child.builder()
-                .party(ChildParty.builder()
-                    .firstName("James")
-                    .detailsHidden(hidden)
-                    .email(EmailAddress.builder().email("email@email.com").build())
-                    .address(Address.builder()
-                        .addressLine1("Address Line 1")
-                        .build())
-                    .telephoneNumber(Telephone.builder().telephoneNumber("01227 831393").build())
-                    .build())
-                .build())
-            .build());
-    }
-
-    private ChildParty getParty(List<Element<Child>> updatedChildren, int i) {
-        return updatedChildren.get(i).getValue().getParty();
-    }
-}
+//package uk.gov.hmcts.reform.fpl.service;
```

Comment:
We can remove all tests in this class now rather than comment them out.. apart from the ones we still use like shouldBuildExpectedLabelWhenEmptyList(), shouldBuildExpectedLabelWhenPopulatedList() and maybe the ones related to addPageShowToCaseDetails ?

Reply:
ahh good spot, shall address this now